.github/workflows/issues.yml

```diff
@@ -21,6 +21,13 @@ jobs:
           # to the issue
           project-url: https://github.com/orgs/zitadel/projects/2
           github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
+      - name: add "thumbs up reaction
+        if: ${{ github.event_name == 'issues' }}
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const { issue: { number: issueNumber }, repository: { owner: { login: owner }, name: repo } } = context.payload;
+            await github.reactions.createForIssue({ owner, repo, issue_number: issueNumber, content: "+1" });
```

Comment:
from what i understand this does an upvote on the issue. in which name is that happening? i guess in my name, as the pat is from me, in that case i think that doesn't make sense

Reply:
I agree, if it's a pat, we should change that